test/python/test_quantumprogram.py

```diff
@@ -1216,6 +1216,7 @@ def test_simulator_online_size(self):
         qc.measure(qr, cr)
         shots = 1  # the number of shots in the experiment.
         QP_program.set_api(QE_TOKEN, QE_URL)
+        qiskit.backends.discover_api_backends(QP_program.get_api())
```

Comment:
As talked, we are going to hide this function to the user into the SDK.

Reply:
I guess this could happen in set_api() since if you're setting the api you'll probably want to use it to access remote backends. There should probably be an SDK method which does this outside of a QuantumProgram instance.

Reply:
Good point. I'd like to call it from set_api().